pkg/notifier/notifier.go

```diff
@@ -27,16 +27,19 @@ type NotifierBuilder interface {
 
 // Represents valid notify states that can be reported
 const (
-	MuoStatePending      MuoState = "StatePending"
-	MuoStateStarted      MuoState = "StateStarted"
-	MuoStateCompleted    MuoState = "StateCompleted"
-	MuoStateDelayed      MuoState = "StateDelayed"
-	MuoStateFailed       MuoState = "StateFailed"
-	MuoStateCancelled    MuoState = "StateCancelled"
-	MuoStateScheduled    MuoState = "StateScheduled"
-	MuoStateScaleSkipped MuoState = "StateScaleSkipped"
-	MuoStateSkipped      MuoState = "StateSkipped"
-	MuoStateHealthCheck  MuoState = "StateHealthCheck"
+	MuoStatePending                       MuoState = "StatePending"
+	MuoStateStarted                       MuoState = "StateStarted"
+	MuoStateCompleted                     MuoState = "StateCompleted"
+	MuoStateDelayed                       MuoState = "StateDelayed"
+	MuoStateFailed                        MuoState = "StateFailed"
+	MuoStateCancelled                     MuoState = "StateCancelled"
+	MuoStateScheduled                     MuoState = "StateScheduled"
+	MuoStateScaleSkipped                  MuoState = "StateScaleSkipped"
+	MuoStateSkipped                       MuoState = "StateSkipped"
+	MuoStateHealthCheck                   MuoState = "StateHealthCheck"
+	MuoStateControlPlaneUpgradeStartedSL  MuoState = "StateControlPlaneStartedSL"
```

Comment:
There are 3 new states introduced here, will validateStateTransition be updated?

Reply:
No. `validateStateTransition` is for the mapping between MUO states and OCM states. But for the notifications we only use MUO states along with SL states so won't require to add it there.

Reply:
Understood